tests/01_plugins/sensors/test_mdistant.py

```diff
@@ -359,3 +359,57 @@ def test_checkerboard(variant_scalar_rgb):
 
     expected = l_o * 0.5 * (rho0 + rho1) / np.pi
     assert np.allclose(result, expected, atol=1e-3)
+
+
+@pytest.mark.parametrize("offset", [0.9, 0.5, 0.1])
+@pytest.mark.parametrize("sigma", [0.1, 0.5, 1.0])
+def test_ray_offset(variant_scalar_rgb, offset, sigma):
```

Comment:
Could you document the idea behind this test? I don't get it :(

Reply:
Fair point, it's straightforward only when you know what's being computed ^^" I'll add a note.